.buildkite/scripts/lifecycle/post_command.sh

```diff
@@ -36,12 +36,15 @@ if [[ "$IS_TEST_EXECUTION_STEP" == "true" ]]; then
   buildkite-agent artifact upload '.es/**/*.hprof'
   buildkite-agent artifact upload 'data/es_debug_*.tar.gz'
 
-  if [[ $BUILDKITE_COMMAND_EXIT_STATUS -ne 0 ]] && \
-      # Skip when triggered from elasticsearch's validation pipeline
-     [[ $BUILDKITE_TRIGGERED_FROM_BUILD_PIPELINE_SLUG != 'elasticsearch-serverless-intake' ]]
-  then
-    echo "--- Run Failed Test Reporter"
-    node scripts/report_failed_tests --build-url="${BUILDKITE_BUILD_URL}#${BUILDKITE_JOB_ID}" 'target/junit/**/*.xml'
+  if [[ $BUILDKITE_COMMAND_EXIT_STATUS -ne 0 ]]; then
+    if [[ $BUILDKITE_TRIGGERED_FROM_BUILD_PIPELINE_SLUG == 'elasticsearch-serverless-intake' ]]; then
+      echo "--- Run Failed Test Reporter (only junit)"
+      node scripts/report_failed_tests --build-url="${BUILDKITE_BUILD_URL}#${BUILDKITE_JOB_ID}" 'target/junit/**/*.xml'\
+        --no-github-update --no-index-errors
```

Comment:
@elastic/kibana-operations - am I understanding these scripts right, that the test failure report is what's missing for me, and to prevent it from muddying our stats (probably the reason why it was skipped before) we can add these flags?

Reply:
I'd give it a test run before merging, I don't think the flags have ever been used.  Sounds right though based on the documentation.

Reply:
Tested: https://buildkite.com/elastic/kibana-elasticsearch-serverless-verify-and-promote/builds/1962 - seems to be working, and does what I hoped! 

However, it's interesting, that breaking one test broke 2 suites, maybe there's some overlap.